src/components/product/my-basket-card/index.tsx

```
@@ -8,6 +8,7 @@ import DeleteProductBtn from '../delete-product';
 import ProductThumbnailImage from '../product-thumbnail';
 import WaitingCounts from '../waiting-counts';
 import * as styles from './index.css';
+import { cardListMypage } from '@/components/product/my-basket-card-list/index.css';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)